api/clan.go

```diff
@@ -799,12 +804,29 @@ func RetrieveClanHandler(app *App) func(c echo.Context) error {
 
 		var clanResult map[string]interface{}
 		err = WithSegment("clan-retrieve", c, func() error {
+			options := models.NewDefaultGetClanDetailsOptions(app.Config)
+			maxApps, err := strconv.ParseUint(maxPendingApplications, 10, 32)
+			if err == nil && int(maxApps) <= options.MaxPendingApplications {
+				options.MaxPendingApplications = int(maxApps)
+			}
+			maxInvs, err := strconv.ParseUint(maxPendingInvites, 10, 32)
+			if err == nil && int(maxInvs) <= options.MaxPendingInvites {
+				options.MaxPendingInvites = int(maxInvs)
+			}
+			if models.IsValidOrder(pendingApplicationsOrder) {
```

Comment:
usually I'd rather return a bad request if the arguments are invalid, instead of swallowing the error (same for max)

Reply:
Done!